.github/check-license.go

```diff
@@ -67,6 +59,12 @@ func checkHeader(path string, header string) bool {
 	}
 	defer f.Close()
 
+	// skip the lines containing the project name and copyright year
+	_, err = f.Seek(int64(len(projectInfo)), io.SeekStart)
+	if err != nil {
+		return false
+	}
+
```

Comment:
can we use regexp or another way to check these two lines?

Reply:
Another way could be to read and discard the first two lines with `io.ReadFull` (though I don't think that's better than this).

With regexp, I think we could use the copyright text as a pattern and allow for any year (something like `\d{4}`), but I can't quite figure how to prepare a multiline regex that `regexp.MustCompile` accepts yet